pandas/tests/indexing/test_scalar.py

```diff
@@ -302,3 +302,36 @@ def test_iat_dont_wrap_object_datetimelike():
         assert result is ser2[1]
         assert isinstance(result, timedelta)
         assert not isinstance(result, Timedelta)
+
+
+def test_tuple_indexed_series_at_get():
+    # GH 26989
+    # Series.at works with MultiIndex
+    series = Series([1, 2], index=[(1, 2), (3, 4)])
+    assert series.at[1, 2] == 1
+
+
+def test_tuple_indexed_series_at_set():
+    # GH 26989
+    # Series.at works with MultiIndex
```

Comment:
same here.  probably should add an assertion about the index type to clarify the difference between these two vs the next two tests